pyflowater/flostream.py

```diff
@@ -0,0 +1,56 @@
+"""Code relating to the firestore API Flo use to stream live data."""
+
+import requests
+import json
+from google.cloud import firestore
+from google.oauth2.credentials import Credentials
+
+from pyflowater.const import (
+    FLO_GOOGLE_API_KEY,
+    FIREBASE_REST_API,
+    FLO_FIRESTORE_PROJECT,
+)
+
+def _get_token_info(token):
+    url = f"{FIREBASE_REST_API}/verifyCustomToken?key={FLO_GOOGLE_API_KEY}"
+    headers = {"Content-type": "application/json; charset=UTF-8"}
+    data = json.dumps({"returnSecureToken": True, "token": token})
+    resp = requests.post(url, headers=headers, data=data)
+    return resp.json()
+
+class FloListener:
+    """Flo firestore listener class."""
+
+    def __init__(self, token, deviceId, callback):
+        self._token = token
```

Comment:
In the docstring, please describe that the callback will be called with however-many-args-it-will-be-called-with (see below), and that its return value if any will be ignored.